workspaces/azure-devops/plugins/scaffolder-backend-module-azure-devops/README.md

```diff
@@ -0,0 +1,5 @@
+# @backstage-community/plugin-scaffolder-backend-module-azure-devops
+
+The azure module for [@backstage/plugin-scaffolder-backend](https://www.npmjs.com/package/@backstage/plugin-scaffolder-backend).
+
+_This plugin was created through the Backstage CLI_
```

Comment:
I missed this last time around, can we get some details on how to install this module as well as some details about the current action - ideally with input and output examples, you can just grab the great examples you have in code 👍

Reply:
Can you help me with the examples? I can't seem to get them to render on the scaffolder actions page. 

<img width="1373" alt="image" src="https://github.com/user-attachments/assets/f9c2feb4-51a8-4429-afe4-1f1d1039be38">

Reply:
I figured it out!